relayer/src/main.rs

```diff
@@ -1,66 +0,0 @@
-// Copyright (c) 2022 Espresso Systems (espressosys.com)
-// This file is part of the Configurable Asset Privacy for Ethereum (CAPE) library.
-
-// This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
-// This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
-// You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
-
-#[macro_use]
-extern crate lazy_static;
-
-use crate::api_server::init_web_server;
-use crate::block_builder::Builder;
-use crate::configuration::{reset_state, store_path};
-use crate::state_persistence::StatePersistence;
-use crate::txn_queue::TxnQueue;
-
-use cap_rust_sandbox::model::{CapeContractState, CAPE_MERKLE_HEIGHT};
-use cap_rust_sandbox::universal_param::verifier_keys;
-use jf_cap::MerkleTree;
-
-use async_std::sync::{Arc, RwLock};
-
-mod api_server;
-mod block_builder;
-mod configuration;
-mod state_persistence;
-mod txn_queue;
-
-#[async_std::main]
-async fn main() -> std::io::Result<()> {
-    tracing_subscriber::fmt()
-        .compact()
-        .with_ansi(false)
-        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
```

Comment:
I checked unused dependencies with cargo udeps and found

```
atomic_store
dirs
tide-websockets
toml
tracing-log
tracing-subscribers
```
The code still compiles without them.

However for the tracing it's because it was set up in`main.rs` and is therefore missing now. We should move this to the minimal relayer binary. But also remove the truly unused dependencies.

Reply:
Thank you for spotting that.
Changes done in d3ede09.